chapters/ch55.md

```diff
@@ -1,78 +1,82 @@
 
 > 2. Try manually modifying B (detect pedestrian)’s output to be the "perfect" output for B. If the algorithm now gives a correct output, then attribute the error to component B. Otherwise, go on to Step 3.
 
---> _replace THIS LINE by your translation for the above line_
+Tương tự, ta thử điều chỉnh thủ công kết quả đầu ra của thành phần B (phát hiện người đi bộ) sao cho "hoàn hảo". Nếu thuật toán cho ra kết quả lái xe tốt, ta có thể quy lỗi cho B. Ngược lại, ta tiếp tục ở bước 3.
 
 > 3. Attribute the error to component C.
 
---> _replace THIS LINE by your translation for the above line_
+Sau khi đã kiểm chứng nguyên nhân phát sinh lỗi không phải do A và B gây ra, ta quy lỗi cho thành phần C. 
 
 > The components of an ML pipeline should be ordered according to a Directed Acyclic Graph (DAG), meaning that you should be able to compute them in some fixed left-to-right order, and later components should depend only on earlier components’ outputs. So long as the mapping of the components to the A->B->C order follows the DAG ordering, then the error analysis will be fine. You might get slightly different results if you swap A and B:
 
---> _replace THIS LINE by your translation for the above line_
+Các thành phần của một mô hình học máy dạng pipeline nên được tổ chức theo lý thuyết đồ thị định hướng không tuần hoàn (DAG), nghĩa là bạn có thể tính toán chúng theo thứ tự cố định từ trái sang phải, và các thành phần sau sẽ phụ thuộc chỉ vào kết quả đầu ra của các thành phần trước đó. Miễn là việc xâu chuỗi các thành phần theo thứ tự A->B->C tuân thủ theo quy tắc đồ thị định hướng không tuần hoàn (DAG), việc phân tích lỗi sẽ tốt. Bạn có thể nhận được các kết quả hơi khác nhau nếu hoán chuyển vị trí của A và B cho nhau như sau:
 
 > A: Detect pedestrians (was previously ​*Detect cars*​)
 
---> _replace THIS LINE by your translation for the above line_
+A: Nhận dạng người đi bộ (trước đây là "Nhận dạng xe")
 
 > B: Detect cars (was previously ​*Detect pedestrians*)​
 
---> _replace THIS LINE by your translation for the above line_
+B: Nhận dạng xe (trước đây là "Nhận dạng người đi bộ")
 
 > C: Plan path for car
 
---> _replace THIS LINE by your translation for the above line_
+C: Hoạch định đường đi cho xe
 
 > But the results of this analysis would still be valid and give good guidance for where to focus your attention.
 
---> _replace THIS LINE by your translation for the above line_
+Tuy nhiên, cho dù các kết quả của việc phân tích lỗi có hơi khác nhau thì chúng vẫn hợp lệ và cho ta định hướng tốt thành phần nào của hệ thống cần cải thiện.
```

Comment:
hơi thừa ý so với bản gốc, mình cũng không rõ có nên không

Reply:
Ừa, mình cũng không chắc nên bỏ hay nên nói rõ ra thì phù hợp cho bản dịch này.  Chỗ này mình tạm thời không sửa. Nếu có yêu cầu từ nhóm, mình sẽ điều chỉnh.

Reply:
@goldenretriever2015 theo mình không nên thêm ý từ bản gốc vào. Người dịch có thể thêm chú thích và tách riêng ra chứ không nên viết thêm các ý mà tác giả không đề cập.

Chúng ta cũng có thể làm như thế nhưng cần thống nhất từ đầu đến giờ. Không nên có một chương hoàn toàn với cách thức làm của các chương còn lại.

Reply:
Mình sẽ theo mục tiêu của nhóm. Mình đã cập nhật lại phần này.